sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamSortRelTest.java

```diff
@@ -92,6 +92,27 @@ public void testOrderBy_basic() throws Exception {
   }
 
   @Test
+  public void testOrderBy_timestamp() throws Exception {
+    String sql = "INSERT INTO SUB_ORDER_RAM(order_id, site_id, price)  SELECT "
+        + " order_id, site_id, price "
```

Comment:
can you add and check `order_time` in the output column list?

Reply:
But the output table is defined in `prepare()` and used by other test cases. A simpler way is to prepare a different output table in my test case. What do you think? @XuMingmin